### PR TITLE
Login and signup backend logic works

### PR DESCRIPTION
**Pull request by sabr5840**
The login and signup function has been created and tested within Postmand.

i have tested the api endpoint: http://localhost:2020/register - with the use of body/JSON and got "Given user details are successfully registrered"

i have tested the api endpoint: http://localhost:2020/dologin - also with the use of body/JSON and got "Du er logget på"

i have tested the same api point just trying the authorization way instead - I got a 403Forbidden...

i dont understand what that means, but something works and I just wanted that to be updated to main. We will discus I morning meeting...

...also all class has been from scratch and renamed - therefor the classes of treatment and booking perhaps has to be changed.

### CODE REVIEW

File: src/main/resources/application.properties
Comment (najamoe) [inline src/main/resources/application.properties:1]: The password dosent match the 1234 we created

**Review by najamoe — CHANGES_REQUESTED** [review]
there are 3 comments that need to be checked out and changes in code before merging

File: src/main/java/com/example/bookingsystembackend/controller/LoginController.java [hunk 1]
```diff
@@ -1,45 +1,70 @@
...
-        // You can return any other information you need
-        return ResponseEntity.ok("Login successful for user: " + userDetails.getUsername());
+    @PostMapping("/dologin")
+    public ResponseEntity<String> doLogin(@RequestBody Customer customer) {
+        Authentication authentication = authenticationManager.authenticate(new UsernamePasswordAuthenticationToken(customer.getEmail(), customer.getPwd()));
+        if(authentication.isAuthenticated()){
+            //return JwtResponseDTO.builder()
+            //        .accessToken(jwtService.GenerateToken(authRequestDTO.getUsername()).build();
+            return ResponseEntity.status(HttpStatus.OK)
+                    .body("Du er logget på");
```
[the remark this answers is not in the document]
Comment (sabr5840) [reply src/main/java/com/example/bookingsystembackend/controller/LoginController.java:64]: yes, now  translated to English (logged in)

File: src/main/resources/application.properties
Comment (sabr5840) [reply src/main/resources/application.properties:1]: dont mind that, it just made it easily for me to work in the workbench. I just out comment it.

**Comment by sabr5840** [comment]
@najamoe @MathildeTrendy please someone review my new changes

**Review by najamoe — APPROVED** [review]

**Comment by najamoe** [comment]
Reviewed and approved
Deleted backend#1 branch

**Pull request merged**